unit_test/environment_test.py

```diff
@@ -28,7 +29,11 @@ def test_inheritance():
 
 
 def test_shell_eval():
-    environment_recipe = parse_environment('VAR="$(echo "a   test" string)"')
+    if sys.platform == "win32":
+        env_string = 'VAR="$(cmd /C "echo a   test string")"'
```

Comment:
I'm wondering why the change in quoting here? The original was deliberately quoted weird to check that the command is invoked with the right arguments

```suggestion
        env_string = 'VAR="$(cmd /C echo "a   test" string)"'
```

Reply:
just chatted with @jack1142 in the discord. `cmd /c echo` actually _adds_ double-quotes into arguments with spaces.

```
Python 3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> subprocess.run(['cmd', '/C', 'echo', 'a string with spaces'])
"a string with spaces"
CompletedProcess(args=['cmd', '/C', 'echo', 'a string with spaces'], returncode=0)
>>> subprocess.run(['cmd', '/C', 'echo', 'a string with spaces', 'word', 'another string with spaces']) 
"a string with spaces" word "another string with spaces"
```

Reply:
we decided to use `python -c 'import sys; print(*sys.argv[1:])'` instead.